hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlOptions.java

```diff
@@ -98,10 +98,20 @@ public static synchronized MysqlOptions instance() {
                     "false"
             );
 
+    public static final ConfigOption<Boolean> AUTO_RECONNECT =
+            new ConfigOption<>(
+                    "jdbc.auto_reconnect",
+                    "Whether to auto reconnect mysql server if it meet " +
```

Comment:
Whether to forced auto reconnect to the database even if the connection fails at the first time.